config/storage.yml

```diff
@@ -1,9 +1,27 @@
+# TODO: remove test, local, and local_public options from this file
+#  figure out how to migrate existing records to handle that change
```

Comment:
ran into some issues with existing database records when I changed `local` to `database` so I left the existing `test`, `local`, and `local_public` here for now and added this TODO. I don't plan to leave this here, but will come back tomorrow and finish. I can either revert the name changes or do a database migration to try to update existing data with the new names. I would definitely want someone with more test data to try the migration if that's the route we take though since I have almost nothing in my db at this point

Reply:
No problem. And for trying things out in development, it really helps to load the fixture data into your dev database. I used to have the seeds file configured to do this but it caused some issues in production. This is an old commit back when it was still working:

https://github.com/AllYourBot/hostedgpt/blob/f66ee3b8d2bac67acfe7f8a174f29930d7176af6/db/seeds.rb

I think this should work as is, although it may be missing a few more recent database tables so check schema.rb and maybe add the missing items to the array. Note: the array is ordered so tables referenced by other tables are loaded first.

Reply:
Oh, after you update this file,  you run `bin/rails db:seed` to load it. If that doesn't work, you maybe have to run `bin/rails db:drop; bin/rails db:prepare`. I vaguely remember having some issues with referential integrity.

And if that still doesn't work, then maybe copy and paste the referential integrity hack from this version of the seeds file:
https://github.com/AllYourBot/hostedgpt/blob/3de36588ca922a22edd232c70fe27967576e78ad/db/seeds.rb

I really probably should have preserved this "load fixtures" trick but moved it into a rake task. If you get this working, maybe when you're done with this PR we tee up another PR that turns this into a clean rake task. The only reason I removed this is because it was causing some issues with the various production hosting environments due to stricter postgres permissions, but this is quite helpful in development.